omnigibson/robots/tiago.py

```diff
@@ -679,9 +675,9 @@ def arm_workspace_range(self):
             "right": [np.deg2rad(-75), np.deg2rad(-15)],
         }
 
-    def get_position_orientation(self):
+    def get_position_orientation(self, clone=True):
         # TODO: Investigate the need for this custom behavior.
-        return self.base_footprint_link.get_position_orientation()
+        return self.base_footprint_link.get_position_orientation(clone=clone)
```

Comment:
Remove clone

Reply:
resolved in person -- keep for now, revert later